UPGRADE.md

```diff
@@ -2,6 +2,45 @@
 
 ## dev-develop
 
+### Definition of security contexts
+
+The definition of security contexts in the `Admin` classes have changed. They
+used to look like the following example:
+
+```php
+public function getSecurityContexts()
+{
+    return [
+        'Sulu' => [
+            'Media' => [
+                'sulu.media.collections',
```

Comment:
i would do because it is not clearly what it is afterwards if you don´t upgrade.